Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [JASP/jaspDesktop](https://hosted.weblate.org/projects/jasp/jaspdesktop/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/jasp/-/jaspdesktop/horizontal-auto.svg)

changyahu committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 78.7% (608 of 772 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/zh_Hant/

shun2wang committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 79.0% (610 of 772 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/zh_Hant/

shun2wang commented on Desktop/po/jaspDesktop-zh_Hant.po:
```diff
@@ -1782,230 +1781,246 @@ msgstr "圖"
-msgstr ""
+msgstr "≤ %1"
 
 #, qt-format
```
thank you for your all effort @changyahu, I don't know if this would be hurt that translator using '< > ≤...' rather than HTML symbol...maybe it's ok?(so I had changed it back on Weblate )

@boutinb  but I think it's better to remove `tr()` [here](https://github.com/jasp-stats/jasp-desktop/blob/583604e1b1b0ebb15602b46014cac0cab0979a9c/QMLComponents/controls/jaspdoublevalidator.cpp#L80-L101) right? it's no sense  translate a symbol and placeholder:-)

changyahu commented:
Hello Shun,

Sure. The update is fine. Thanks for checking.

Changya Hu


On Mon, Nov 7, 2022 at 10:43 AM Shun Wang ***@***.***> wrote:

> ***@***.**** commented on this pull request.
> ------------------------------
>
> In Desktop/po/jaspDesktop-zh_Hant.po
> <https://github.com/jasp-stats/jasp-desktop/pull/4908#discussion_r1014958966>
> :
>
> >  msgctxt "JASPDoubleValidator|"
>
>  msgid "&#8804; %1"
>
> -msgstr ""
>
> +msgstr "≤ %1"
>
>
>
>  #, qt-format
>
>
> thank you for your all effort @changyahu <https://github.com/changyahu>,
> I don't know if this would be hurt that translator using '< > ≤...' rather
> than HTML symbol...(so I had changed it back on Weblate )
>
> @boutinb <https://github.com/boutinb> but I think it's better to remove
> tr() here
> <https://github.com/jasp-stats/jasp-desktop/blob/583604e1b1b0ebb15602b46014cac0cab0979a9c/QMLComponents/controls/jaspdoublevalidator.cpp#L80-L101>
> right? it's no sense translate a symbol and placeholder:-)
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jasp-stats/jasp-desktop/pull/4908#pullrequestreview-1169731985>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ANYKI5ZYL65IV6VSZNCX5ADWHBUFTANCNFSM6AAAAAARYXBP6E>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>

changyahu committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 79.0% (610 of 772 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/zh_Hant/

changyahu committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 79.0% (610 of 772 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/zh_Hant/

tmisiuro committed:
Added translation using Weblate (Polish)

tmisiuro committed:
Translated using Weblate (Polish)

Currently translated at 6.0% (50 of 825 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/pl/

SCgeeker committed:
Translated using Weblate (Chinese (Traditional))

Currently translated at 93.1% (719 of 772 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/zh_Hant/

tmisiuro committed:
Translated using Weblate (Polish)

Currently translated at 37.5% (310 of 825 strings)

Translation: JASP/jaspDesktop
Translate-URL: https://hosted.weblate.org/projects/jasp/jaspdesktop/pl/
